src/webpimage.cpp

```diff
@@ -508,7 +509,8 @@ namespace Exiv2 {
         while ( !io_->eof() && (uint64_t) io_->tell() < filesize) {
             io_->read(chunkId.pData_, WEBP_TAG_SIZE);
             io_->read(size_buff, WEBP_TAG_SIZE);
-            long size = Exiv2::getULong(size_buff, littleEndian);
+            const long size = Exiv2::getULong(size_buff, littleEndian);
+            EXIV2_ENFORCE(size < filesize - io_->tell());
```

Comment:
Tiny suggestion. I would write this code as:

```
EXIV2_ENFORCE(size < (filesize - io_->tell()));
```
It's always clearer for me being explicit about the operators than relying in the operator precedence rules.